db/table_cache.cc

```diff
@@ -336,10 +337,21 @@ Status TableCache::Get(const ReadOptions& options,
 
     if (auto row_handle =
             ioptions_.row_cache->Lookup(row_cache_key.GetUserKey())) {
+
+      // Cleanable routine to release the cache entry
+      Cleanable cache_entry_clean;
```

Comment:
Can you make sure to consider this comment? https://github.com/facebook/rocksdb/pull/2492#discussion_r124064384